src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Quadrangle {
+    public IsoscelesTrapezoid(String color, int width, int height) {
+        super(color, width, height);
+    }
+
+    @Override
+    public double getArea() {
+        return super.getArea();
+    }
+
+    @Override
+    public String draw() {
+        return "Figure: isoscelesTrapezoid" + super.draw();
```

Comment:
The `draw()` method should provide specific information about the object being drawn. Consider including the dimensions, color, and area of the trapezoid in the returned string.